### DISCUSSION TITLE
question about custom location

### DISCUSSION THREAD

**Discussion by RaiRoo** [NginxProxyManager/nginx-proxy-manager#2604] [Q&A]
All my hosts/services are like http://192.168.178.abc:<portnumber> and are working fine...

But now I want a specific hostname - say x.y.com - point to the destination http://192.168.178.121:8082/iqontrol/index.html. 
This should be possible with a custom location, but I have no idea how this can be done.

**Comment by viktak** [comment]
I would also like to learn this... Have you figured it in themeantime?
Also, is it possible to use an external host as a destination?
